my-dsl.md

```diff
@@ -1,38 +1,60 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
-
+The name of the language is EasyLanguage. There is documentation
+here: http://www.tradestation.com/~/media/Files/TradeStation/Education/University/School%20of%20EasyLanguage/Books/EL_Essentials.ashx
 
 # Domain
 _Describe the language's domain in five words._
-
+Financial traders who use TradeStation
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+EasyLanguage is compiled to machine code, and then the compiled code is 
+executed. 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+EasyLanguage is designed for people who are not neccessarily programmers but who 
+need to make complex statements regarding stock trades. It allows for creating 
+functions and it is compiled to machine code. However, users aren't necessarily
+ expected to write functions. When used in the way it is intended, EasyLanguage 
+ is much simpler than a full language might be. and has very "englishy" syntax. 
+ Because that syntax is so clearly intended for the almost-exclusive purpose of 
+ finance-related actions, I would argue that it is closer to a pure DSL than a 
+ general purpose language.
 
```

Comment:
I would definitely agree with your assessment of the DSL-ness of this language. Even though it might be able to create other programs, it seems very focused on allowing financial investors who have little to no programming knowledge to make programs easily. It definitely seems domain focused and has a unique syntax ( two properties we discussed in "what makes a DSL ).